Detectors/TPC/reconstruction/include/TPCReconstruction/TPCFastTransformHelperO2.h

```diff
@@ -89,7 +89,7 @@ class TPCFastTransformHelperO2
   std::unique_ptr<TPCFastTransform> create(Long_t TimeStamp);
 
   /// Updates the transformation with the new time stamp
-  int updateCalibration(TPCFastTransform& transform, Long_t TimeStamp);
+  int updateCalibration(TPCFastTransform& transform, Long_t TimeStamp, float vDriftFactor = 1.f);
```

Comment:
I added a vdrift scaling factor here.
In the long run, I think it is best to pass in the run/time dependent calibration values as arguments to this function instead of querying them inside the `TPCFastTransformHelperO2::updateCalibration` function, so I would propose we just extend this further in the future.